wireguard-go/README.md

```diff
@@ -0,0 +1,39 @@
+This directory contains to patches that can be applied to wireguard-go to leverage wolfCrypt as a crypto engine.
```

Comment:
Note that this requires Go > 1.20, which is newer than the version in the Ubuntu 22.04 repos. So many users will hit that error and need to manually install a newer Go version